data/brands/shop/gift.json

```diff
@@ -220,6 +220,8 @@
       "locationSet": {"include": ["be", "nl"]},
       "tags": {
         "brand": "Wereldwinkel",
+        "brand:wikidata": "Q1640782", 
+        "brand:wikipedia": "en:Wereldwinkel",
```

Comment:
Did you mean to use the "_nl_" version of Wikipedia in this edit, for example: "brand:wikipedia": "**nl**:Wereldwinkel", as the English version doesn't seem to exist.